pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -108,7 +151,9 @@ func (s *NodeController) Run(period time.Duration, syncNodeList, syncNodeStatus
 			glog.Errorf("Error loading initial static nodes: %v", err)
 		}
 	}
+	nodes = s.DoChecks(nodes)
 	nodes, err = s.PopulateAddresses(nodes)
+	nodes = reconcilePodCIDRs(nodes, &api.NodeList{})
```

Comment:
Is there a reason we need to do this both in Run() and in SyncCloud()? It would be nice if we could at least keep this CIDR-distribution stuff isolated in the in-the-cloud section.